src/pkg/cli/client/byoc/do/byoc.go

```diff
@@ -148,8 +148,7 @@ func (b *ByocDo) deploy(ctx context.Context, req *defangv1.DeployRequest, cmd st
 
 	etag := pkg.RandomID()
 
-	serviceInfos := []*defangv1.ServiceInfo{}
-
+	var serviceInfos []*defangv1.ServiceInfo
```

Comment:
Just curious, what's the motivation for this change?

Reply:
Making it identical to the other BYOC files. I'll remove it.

Reply:
No need to remove it, just wanted to understand the purpose.